tests/Form/InputUrlTest.php

```
@@ -0,0 +1,610 @@
+
+    public function testRenderWithValue(): void
+    {
+        self::assertSame(
+            <<<HTML
+            <input id="inputurl-" type="url" value="https://example.com">
+            HTML,
+            InputUrl::tag()->id('inputurl-')->value('https://example.com')->render(),
+            "Failed asserting that element renders correctly with 'value' attribute.",
+        );
+    }
+}
```

PHPMD (2.15.0)

[warning] The class InputUrlTest has 51 public methods and attributes. Consider reducing the number of public items to less than 45. (undefined)

(ExcessivePublicCount)